egs/iam/s5/local/add_test_data_to_train.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
```

Comment:
This script is not needed really. 
You can add a few lines to run.sh to do this:
Get the set of unique words in test text by `cat data/test/text | awk '{ for(i=2;i<=NF;i++) print $i;}' | sort -u >test_words.txt`. Do this for train text too.
Subtract train words from test words: `filter_scp.pl --exclude train_words.txt test_words.txt >diff.txt`
Append them to train text: `cat diff.txt | awk '{ print "id " $1 }' >> train/text`

Actually, we won't need this type of LM altogether after we adopt a standard LM like the one in that paper you mentioned.